cmd/client/util.go

```diff
@@ -125,74 +125,70 @@ func parseAttributes(rootArgs *rootArgs) (*mixerpb.Attributes, error) {
 
 	// the following boilerplate would be more succinct with generics...
 
-	if m, err := process(attrs.Dictionary, rootArgs.stringAttributes, parseString); err != nil {
+	var m map[int32]interface{}
+	var err error
+
+	if m, err = process(attrs.Dictionary, rootArgs.stringAttributes, parseString); err != nil {
```

Comment:
Was this complaining about "shadowing" ?

Can you put a link to the lint log here?